gridt/src/app/core/lastSignal.model.ts

```diff
@@ -0,0 +1,8 @@
+export interface LastSiganl {
+
```

Comment:
Spelling mistake

Reply:
Could you call this just "signal"? It will be used in other contexts than just the last signal.